src/dbus_api/pool/pool_3_0/methods.rs

```diff
@@ -56,13 +56,17 @@ pub fn create_filesystems(m: &MethodInfo<MTSync<TData>, TData>) -> MethodResult
     let mut mutex_lock = dbus_context.engine.blocking_lock();
     let (pool_name, pool) = get_mut_pool!(mutex_lock; pool_uuid; default_return; return_message);
 
-    let result = log_action!(pool.create_filesystems(
-        &pool_name,
-        pool_uuid,
-        &filesystems
-            .map(|x| (x, None))
-            .collect::<Vec<(&str, Option<Sectors>)>>(),
-    ));
+    let result = handle_action!(
```

Comment:
My thought here is that if a filesystem method ever causes the thin pool state to change to failing, this could eventually change the pool state and so we want to catch any error that could result in a pool state change. This is mostly future proofing for when we take further advantage of pool states and error chaining. Does that sound reasonable to you?

Reply:
Yup!